packages/main/src/Toast.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="ui5-toast-root-parent"
+	dir="{{rtl}}">
+	<div class="ui5-toast-root"
+		role="alert"
+		style="{{styles.root}}"
+		@transitionend="{{_ontransitionend}}">
+		<span class="ui5-toast-text">
+			<bdi>
+				<slot></slot>
+			</bdi>
+		</span>
+	</div>
```

Comment:
can we flatten at least one level of this?
Possible improvements:
- Remove at least 1 div
- do we need the span?

Reply:
- Can't remove any of the divs, since it'll ruin vertical/horizontal alignment concept.
- The span is needed in order to properly style the text, which exists in the Toast

Reply:
We agreed upon keeping the current DOM hierarchy.